### PR TITLE
Additions to metaitem bracket handler & /gt util hand changes 

### PR DESCRIPTION
**Pull request by brachy84**
- `/gt util hand` will no longer work. The new command is `/gt hand`. The old one will show a msg to use the new one
- the metaitem name when using the command is colored
- by hovering over the name it shows that it has been copied to clipboard
- added ore dicts to the command for any item
- removed translation for command output 

- compressed blocks can now be obained with `<metaitem:blockMaterial>` in ct
- frames can now be obained with `<metaitem:frameMaterial>` in ct
- pipes and cables can now be obained with `<metaitem:pipeOreDictMaterial>` in ct (example: `<metaitem:wireGtSingleCopper>`)
- mte's can be obtained with `<metaitem: id >` (`meta_tile_entity` still works)

### CODE REVIEW

File: src/main/java/gregtech/common/command/CommandHand.java [hunk 1]
```diff
@@ -82,27 +89,78 @@ public void execute(@Nonnull MinecraftServer server, @Nonnull ICommandSender sen
...
+                    player.sendMessage(new TextComponentString("MetaItem Id: ").appendSibling(new TextComponentString(id).setStyle(new Style().setColor(TextFormatting.GREEN)))
+                            .setStyle(getCopyInfoStyle(ctId)));
+                }
+            } else {
+                Block block = Block.getBlockFromItem(stackInHand.getItem());
+                String id = null;
+                if (block instanceof BlockCompressed) {
+                    id = "block" + ((BlockCompressed) block).getGtMaterial(stackInHand.getMetadata()).toCamelCaseString();
+                } else if (block instanceof BlockFrame) {
+                    id = "block" + ((BlockFrame) block).getGtMaterial(stackInHand.getMetadata()).toCamelCaseString();
```
Comment (ALongStringOfNumbers) [inline src/main/java/gregtech/common/command/CommandHand.java:121]: frame?

File: src/main/java/gregtech/common/blocks/FrameItemBlock.java [hunk 2]
```diff
@@ -33,4 +37,12 @@ public String getItemStackDisplayName(@Nonnull ItemStack stack) {
         Material material = getBlockState(stack).getValue(frameBlock.variantProperty);
         return OrePrefix.frameGt.getLocalNameForItem(material);
     }
+
+    @Override
+    public void addInformation(ItemStack stack, @Nullable World worldIn, List<String> tooltip, ITooltipFlag flagIn) {
+        super.addInformation(stack, worldIn, tooltip, flagIn);
+        if (flagIn.isAdvanced()) {
+            tooltip.add("MetaItem Id: block" + frameBlock.getGtMaterial(stack.getMetadata()).toCamelCaseString());
```
Comment (ALongStringOfNumbers) [inline src/main/java/gregtech/common/blocks/FrameItemBlock.java:45]: frame?

**Review by ALongStringOfNumbers — CHANGES_REQUESTED** [review]
The frames have the meta ids of blocks. EG the Invar frame has the meta id of `blockInvar`, while the invar block also has this meta id. This is both in the advanced information and in the output of the command.

After copying the oredict name to the clipboard, you cannot copy the meta id name to clipboard. Clicking on the name does nothing. In addition, the hover comment does not update, meaning that the comment "Click to copy (oredict name) to clipboard" still shows up on hover, and "(meta id name) was copied to clipboard still shows up when hovering over the meta name.

File: src/main/java/gregtech/common/blocks/FrameItemBlock.java (hunk 2 above)
Comment (brachy84) [reply src/main/java/gregtech/common/blocks/FrameItemBlock.java:45]: yeah, i noticed that after looking over it. Will change it later

**Comment by brachy84** [comment]
i don't think there is a way to dynamically change the hovering text

**Review by ALongStringOfNumbers — APPROVED** [review]

**Pull request merged**